test/e2e/set/scyllacluster/verify.go

```diff
@@ -116,13 +118,32 @@ func verifyScyllaCluster(ctx context.Context, kubeClient kubernetes.Interface, s
 
 	verifyPersistentVolumeClaims(ctx, kubeClient.CoreV1(), sc)
 
-	// TODO: Use scylla client to check at least "UN"
 	scyllaClient, hosts, err := utils.GetScyllaClient(ctx, kubeClient.CoreV1(), sc)
 	o.Expect(err).NotTo(o.HaveOccurred())
 	defer scyllaClient.Close()
 
 	o.Expect(hosts).To(o.HaveLen(memberCount))
 
+	// Because of Gossip issues node status is not consistent across different scylla nodes.
```

Comment:
This is misleading. The necessity of waiting explicitly is not caused by a gossip issue - status propagation is an inherent property of the algorithm. So the only fix for it would be to either take it into account when declaring readiness or to get rid of gossip altogether.

Reply:
What i mean here is that the issue is with the database using gossip whether it's its inherent property or an effect of a particular usage. The state of nodes should behave like a transactional table and when the nodes have quorum the nodes state should be the same for a particular node, no matter who you ask.

Reply:
gone